docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input properties in TextField
+
+Input properties, such as `InputColor` and `InputPointSize`, are used when you want to set new input texts differently from the existing text settings when the new text inputs.

```

Comment:
Recommend rephrase, please verify if this is correct here?
To change the text settings for new input text, you can use the Input properties, such as `InputColor`  and `InputPointSize` of the [Tizen.NUI.BaseComponents.TextField](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html) class.

Reply:
I think it's better. Changed as your suggestion.